### ISSUE TITLE
I dont can understand function getRateDamage in EntityDamageByEntityEvent.

### ISSUE THREAD

**Issue by mixailenko5** [iTXTech/Genisys#1365]
I dont understand this function. I tried to get its value. But i had zero (0) only. Who can tell me about this function and describe about RateDamage.?

**Comment by dog194** [comment]
private $modifiers;------setDamage------getDamage
	private $ratemodifiers = [];------setRateDamage------getRateDamage
Then see getFinalDamage()
and serch setRateDamage in EntityDamageEvent.php.

**Comment by mixailenko5** [comment]
@dog194 no. I can not understand what it is works. And means of this function. Can you describe it?
Rate means - intensity?
